src/helpers/keycard.ts

```diff
@@ -7,15 +7,18 @@ const keycard = new Keycard({
   URL: process.env.KEYCARD_URL || 'https://keycard.snapshot.org'
 });
 
-const verifyKeyCard = async (req, res, next) => {
-  const key = req.headers['x-api-key'] || req.query.apiKey;
-  if (key && keycard.configured) {
-    const { valid, rateLimited } = keycard.logReq(key);
+const checkKeycard = async (req, res, next) => {
+  const apiKey = req.headers['x-api-key'] || req.query.apiKey;
+  if (apiKey && keycard.configured) {
+    const keycardData = keycard.logReq(apiKey);
+    if (!keycardData.valid) return sendError(res, 'invalid api key', 401);
 
-    if (!valid) return sendError(res, 'invalid api key', 401);
-    if (rateLimited) return sendError(res, 'too many requests', 429);
+    res.locals.keycardData = keycardData;
+    res.set('X-Api-Key-Limit', keycardData.limit);
+    res.set('X-Api-Key-Remaining', keycardData.remaining);
+    res.set('X-Api-Key-Reset', keycardData.reset);
```

Comment:
These lint errors should be fixed once we install new version of keycard.js

Reply:
Let's merge and publish the new keycard.js version first, can not even `yarn dev` because of typescript error

Reply:
Using keycard.js from github for now. will use from npm once it is published 

But it is not working on this PR because it is private repo 😃